views/update/update-check-your-answers.html

```diff
@@ -124,23 +124,18 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
         {% include "includes/check-your-answers/relevant-period-check-your-answers.html" %}
       {% endif %}
 
-      {% include "includes/check-your-answers/presenter.html" %}
+      {% if pageParams.isRPStatementExists %}
+      <br>
```

Comment:
Sonarqube does not check the html template files for correct indentation, although this <br> here and include statement below on line 192 would be more consistent with other templates and the code easier to maintain, if indentation were added.